Makefile

```diff
@@ -158,11 +158,18 @@ deep-clean: clean
 	rm -Rdf .ethereumtest/StatusChain
 
 vendor-check:
+	./_assets/patches/patcher -c
+	./_assets/ci/isolate-vendor-check.sh
+
+dep-ensure:
```

Comment:
Any reason why we're not including these targets in the help screen of Make so people are aware of them when they type `make`? We could have patches and dependencies sections. What do you think @mandrigin?

Reply:
I think we should definitely include them